src/main/resources/application.properties

```diff
@@ -16,6 +16,7 @@ terraform.root.module.directory=
 clean.workspace.after.deployment.enabled=true
 spring.retry.max-attempts=3
 spring.retry.delay-millions=1000
+support.default.terraform.versions.only=true
 terraform.install.dir=/opt/terraform
 terraform.download.base.url=https://releases.hashicorp.com/terraform
 terraform.versions=1.6.0,1.7.0,1.8.0,1.9.0
```

Comment:
@baixinsui how about renaming this property to 'default.supported.terraform.versions' ?